astropy/units/astrophys.py

```diff
@@ -54,6 +54,15 @@
 def_unit(['solMass', 'M_sun', 'Msun'], _si.M_sun.value * si.kg, namespace=_ns,
          prefixes=True, doc="Solar mass",
          format={'latex': r'M_{\odot}', 'unicode': 'M⊙'})
+def_unit(['jupiterMass', 'M_jup', 'Mjup','M_jupiter', 'Mjupiter'],
+         _si.M_jup.value * si.kg, namespace=_ns,
+         prefixes=True, doc="Jupiter mass",
+         # LaTeX jupiter symbol requires wasysym
+         format={'latex': r'M_{Jup}', 'unicode': 'M♃'})
```

Comment:
The latex form should be `r'M_{\rm Jup}'`. Actually, should it be `r'M_{\rm J}'`? I think that is used much more commonly (even in the IAU definition in [1])

[1] http://asa.usno.navy.mil/static/files/2014/Astronomical_Constants_2014.pdf
(note this also defines the moon-to-earth mass ratio).
Also http://maia.usno.navy.mil/NSFA/NSFA_cbe.html#MSMJ2009